hack/goreleaser.sh

```diff
@@ -107,11 +108,24 @@ build_single() {
     -w /go/src/github.com/kubeshop/botkube \
     -e GORELEASER_CURRENT_TAG=${GORELEASER_CURRENT_TAG} \
     -e ANALYTICS_API_KEY="${ANALYTICS_API_KEY}" \
+    -e SLACK_APP_TOKEN="${SLACK_APP_TOKEN}" \
     goreleaser/goreleaser build --single-target --rm-dist --snapshot --id botkube -o "./botkube"
   docker build -f "$PWD/build/Dockerfile" --platform "${IMAGE_PLATFORM}" -t "${IMAGE_REPOSITORY}:${GORELEASER_CURRENT_TAG}" .
   rm "$PWD/botkube"
 }
 
+build_test_single() {
+  export GORELEASER_CURRENT_TAG=v9.99.9-dev
+  docker run --rm --privileged \
+    -v "$PWD":/go/src/github.com/kubeshop/botkube \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -w /go/src/github.com/kubeshop/botkube \
+    -e GORELEASER_CURRENT_TAG=${GORELEASER_CURRENT_TAG} \
+    goreleaser/goreleaser build --single-target --rm-dist --snapshot --id botkube-test -o "./botkube-e2e.test"
+  docker build -f "$PWD/build/test.Dockerfile" --platform "${IMAGE_PLATFORM}" -t "${TEST_IMAGE_REPOSITORY}:${GORELEASER_CURRENT_TAG}" --build-arg TEST_NAME=botkube-e2e.test .
+  rm "$PWD/botkube-test"
```

Comment:
Please add the `botkube-test` to the `.gitignore` file. I know that you remove it here, but in case of failed `docker build` it won't be removed and can be commited by mistake.